book/website/ethical-research/data-hazards/dh-case-study.md

```diff
@@ -0,0 +1,281 @@
+- [**created a final volunteer's task form**](), so that all volunteers had been assigned tasks before the event began 
+
+
+(er-datahazardscasestudy-organising-catering)=
+### Catering 
+For our event we choose a local, entirely vegan caterer, to align with the core values of the event. 
+To ensure successful ordering and delivery of the catering on the day we: 
+- estimated the number of attendees to provide catering - including volunteers, speakers and organisers. 
+- recieved a quote from the caterers in advance and ensure we had the budget to cover the quote.
+- collected dietary requirements to ensure that all attendees were catered for, and ensured the catering company recieved these requirements in advance. 
+- arranged a delivery time and location.
+- ensured payment to catering company as per company and organisation policies.
+
+Tip: make sure terms and conditions are read and timing requirements from different parties involved are clear from the beginning. It might be that the caterer you are ordering from requires a purchase order to be completed a certain time in order to complete the order action.
+
+
+(er-datahazardscasestudy-hosting)=
+## Hosting the Event 
+There is already a chapter on {ref}`cl-organising-conferences` which includes an in-depth dive into requirements for accessibility. 
+Likewise there is also a section for {ref}`cm-comms-overview-accessibly`.
+As well as on {ref}`cl-hybrid-collab-guidelines`. 
+
+From our experience, here are some extra tips on the day of the event.
+
+(er-datahazardscasestudy-hosting-startingevent)=
+### Starting the Event 
+- As the hosts arrive in-person at least 1 hour prior to start time to ensure the zoom link and audio-visual equipment (speakers, screens and microphones) are all working.
+- Open the online call early to allow for: 
+    - volunteers to settle in and open any breakout-rooms.
+    - participants to join in advance and set-up break out rooms for networking events and workshops later in the day.
+
+(er-datahazardscasestudy-hosting-hybridevent)=
+### Hybrid Event Tips
+We chose to host the Symposium in a hybrid format as this would allow attendance for those interested in the topic, and speakers who would be unable to attend in-person.
+
+Thanks to the good communication with volunteers and the preparation of the room before the event, our hybrid format ran successfully without hiccups, even though this aspect of the event was challenging due to the number of tasks involved. Here we share some tips that helped us!
+
+To ensure a smooth hybrid event, we:
+- tested the physical room audio-visual the day before and the day of the event, to ensure there were multiple microphones and the capability for live Zoom recordings.
+- had in-person session chairs to introduce speakers and make sure people (both online and in-person) kept to agenda timings.
+- made sure session chairs were aware of the hybrid set-up, including microphones, speakers and Zoom settings,
```

Comment:
Zoom is variously capitalised and not in this document. Ensure it's always capitalised (like it is here)